### ISSUE TITLE
Header aria label not using accessible name

### ISSUE THREAD

**Issue by Dawdre** [alphagov/govuk-frontend#1952]
Currently the [header](https://design-system.service.gov.uk/components/header/) component uses "Menu" as it's button text. Yet the aria label reads "Show or hide Top Level Navigation". It was noted in an external accessibility audit carried out on one of our applications. Causing potential confusion for voice control assistive technology.

This does not comply with WCAG Level A 2.1:
2.5.3 Label in Name – https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html

**Comment by 36degrees** [comment]
Hi @Dawdre,

This is a duplicate of #1937 which has been fixed, but has not yet been released. It will be included in GOV.UK Frontend v3.9.0, which should be released in the next few days.
